SimConnectMSFS/SimConnectCache.cs

```diff
@@ -223,8 +223,6 @@ internal void Start()
 
         private void SimConnectCache_OnRecvClientData(SimConnect sender, SIMCONNECT_RECV_CLIENT_DATA data)
         {
-            
```

Comment:
I have no idea why this sneaked in, because I review all files individually.

Reply:
Oops